### PR TITLE
Added readme for instructions

### PR DESCRIPTION
**Pull request by jbrooksuk**
Essentially what @jskinner put in the forum announcement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Sublime Packages
+
+**Note that these packages will not work with any version of Sublime Text older than dev build 3092.**
```
Comment (guillermooo) [inline README.md]: Perhaps link to the beta download page here?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Installation
+
+If you wish to make any changes to these and test them locally, the simplest way is to fork the the repository, and then symlink the package(s) of interest into your Packages folder, e.g., on OSX:
```
Comment (guillermooo) [inline README.md]: the the => the

File: README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+```bash
+$ git clone https://github.com/sublimehq/Packages.git
+$ ln -s `pwd`/Packages/Python ~/Library/Application\ Support/Sublime\ Text\ 3/Packages/Python
+```
+
+After you've finished, keep in mind that you're now overriding a default package, and when Sublime Text updates to a new version, unless you pull the latest changes from the repository, you'll run the risk of having an out of date package.
```
Comment (guillermooo) [inline README.md]: out-of-date? (adj)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+On Windows, rather than fiddling around with symlinks, the simplest approach would likely be to download the Portable version, and clone the repository directly as a subdirectory of 'Data'.
```
Comment (guillermooo) [inline README.md]: Data (not 'Data'), same as Packages earlier. Or the other way around.
Portable => portable

**Comment by guillermooo** [comment]
Did some reviewing!

File: README.md (hunk 1 above)
Comment (pradyunsg) [inline README.md]: And possibly in a block quote for more visibility.

File: README.md (hunk 4 above)
Comment (jbrooksuk) [inline README.md]: `Data` would be better.

**Comment by jbrooksuk** [comment]
All fixed up.

**Pull request merged**